src/components/SearchName/Search.js

```diff
@@ -85,7 +85,7 @@ function Search({ history, className, style }) {
       action="#"
       onSubmit={e => {
         e.preventDefault()
-        const searchTerm = input.value
+        const searchTerm = input.value.toLowerCase()
```

Comment:
Is this the only place we have to lower case? No need to do it at `validateName` to have similar logics in one place?

Reply:
Validation happens after on the actual search page where we show the error (or the name page if not). The problem right now is we need to at least show the correct lowercase name before the validation happens. If we use validateName here it will throw if it isn't a correct name and never make it to the actual name page.